pkg/cmd/properties/properties.go

```diff
@@ -291,6 +291,13 @@ func (p *properties) DurationPropertyValue(name string) time.Duration {
 	s := p.parseStringValue(name)
 	d, _ := time.ParseDuration(s)
 
+	// Get seconds duration value and check if duration is less than 30s
+	if d.Seconds() < 30 {
+		log.Warnf("config %s has been reset from %s to the lower limit duration of 30s. Please update this value accordingly in your configs", name, s)
+		// Set duration to the lower limit of 30s
+		d, _ = time.ParseDuration("30s")
```

Comment:
Instead of setting to lower limit may be we could use the default values.

Reply:
We would only set if it's below 30 seconds right? So if default is 60s, we leave it alone. I didn't change any defaults. Should I look at any defaults and change to 30s? This only looks at what the users set in their config. Or maybe I'm missing something.